framework/classes/attachment.php

```diff
@@ -60,7 +65,7 @@ public function __construct($attached, $config)
         }
 
         if (!empty($config['image']) && empty($config['extensions'])) {
-            $config['extensions'] = array('jpg', 'gif', 'png', 'jpeg');
+            $config['extensions'] = static::$image_extensions;
             unset($config['image']);
         }
```

Comment:
Yous hould probably lowercase the config too `array_map(array('\Str', 'lower'), static::$image_extensions)`